source/patterns/04-pages/homepage/homepage~ja.yaml

```diff
@@ -1,5 +1,11 @@
 lang: ja
 
+siteHeader:
+  logo:
+    href: '#'
+    image:
+      alt: 'Liberoのロゴ'
```

Comment:
Are these rogue characters in the alt?

Reply:
This is the Google Translate translation of "Libero logo" into Japanese.